module-2/backend/week_one.md

```diff
@@ -5,30 +5,63 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+    * get: receive from server
+    * post: send to server
+    * put: for uploading
+    * delete: Deletes stuff
+    * patch: not sure, read about it in mod 1

```

Comment:
I would say that all of these send and receive from a server (even when you send a POST request, the server sends back some sort of response to say that it's been successful), but GET generally displays information about a resource, while POST is used to create new resources. PATCH/PUT are used to update a resource.